### PR TITLE
fix: color of linear progress track in dark mode

### PR DESCRIPTION
**Pull request by mengxi-ream**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

**Comment by mengxi-ream** [comment]
Hi @xile611 , I've done the most parts (maybe haha), thank you for your help!

But I still have some questions:
1. I chose `'#3A3D42` as the track color for the dark mode but I'm not quite sure if there is any better color theory to help me to choose the better color for the track in dark mode?
2. Apart from the built in dark and light theme, should I change any other theme. For example, change some code in [vchart-theme](https://github.com/VisActor/vchart-theme)?

**Comment by mengxi-ream** [comment]
https://github.com/user-attachments/assets/450f6e06-2582-4d37-a266-f7e9479d2570

The current effects in the dark theme are displayed.

File: packages/vchart/src/theme/builtin/dark/color-scheme.ts [hunk 1]
```diff
@@ -76,7 +76,10 @@ export const colorScheme: IThemeColorScheme = {
       discreteLegendPagerHandlerDisableColor: '#55595F',
 
       /** 占位圆颜色 */
-      emptyCircleColor: '#bbbdc3'
+      emptyCircleColor: '#bbbdc3',
+
+      /** 线性进度条颜色 */
+      linearProgressTrackColor: '#3A3D42'
```
Comment (xile611) [inline packages/vchart/src/theme/builtin/dark/color-scheme.ts:82]: 按照我们设计稿，应该是这个颜色：  `#202226`

Comment (mengxi-ream) [reply packages/vchart/src/theme/builtin/dark/color-scheme.ts:82]: Get！咱们是有专门的设计师来弄设计稿吗？

Comment (mengxi-ream) [reply packages/vchart/src/theme/builtin/dark/color-scheme.ts:82]: 如果是换成 `#202226` 会和默认的 dark mode 的背景色一致，是不是不太对啊？

https://github.com/user-attachments/assets/ff725dac-1175-4213-b43c-b02ed37545a0

Comment (xile611) [reply packages/vchart/src/theme/builtin/dark/color-scheme.ts:82]: @mengxi-ream 用这个颜色： `#404349`
 有设计师设计的，不过也不是很细

image

Comment (mengxi-ream) [reply packages/vchart/src/theme/builtin/dark/color-scheme.ts]: done

**Review by xile611 — APPROVED** [review]

**Pull request merged**